### PR TITLE
Change settings for weekly CI build job

### PR DESCRIPTION
**Pull request by bantonsson**

### CODE REVIEW

File: dd-java-agent/instrumentation/netty-4.1/netty-4.1.gradle [hunk 1]
```diff
@@ -17,6 +17,7 @@ muzzle {
     module = "netty-all"
     versions = "[4.1.0.Final,)"
     assertInverse = true
+    excludeDependency 'io.netty:netty-tcnative'
```
Comment (devinsba) [inline dd-java-agent/instrumentation/netty-4.1/netty-4.1.gradle:20]: Hmm, what is going on here?

**Review by devinsba — APPROVED** [review]

File: dd-java-agent/instrumentation/netty-4.1/netty-4.1.gradle (hunk 1 above)
Comment (bantonsson) [reply dd-java-agent/instrumentation/netty-4.1/netty-4.1.gradle:20]: Weird dependencies in the pom files `netty-tcnative-2.0.39.Final-${os.detected.classifier}.jar` does not play well with `gradle`. Maybe the `io.netty-netty-all-4.1.64.Final` release is broken and we should skip it instead?

**Review by richardstartin — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -101,13 +102,15 @@ jobs:
 
   build_clean_cache:
     <<: *defaults
+    resource_class: xlarge
```
Comment (tylerbenson) [inline .circleci/config.yml:105]: oops... sorry I missed this.

**Review by tylerbenson — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (bantonsson) [reply .circleci/config.yml:105]: No worries. It still failed until I capped the maven heap.

**Pull request merged**